src/main/java/gregtech/loaders/recipe/handlers/ToolRecipeHandler.java

```diff
@@ -448,44 +436,46 @@ public static void registerManualToolRecipes() {
     }
 
     private static void registerFlintToolRecipes() {
+        /*
```

Comment:
Is this still going to be worked on?

Reply:
Appears like this enchantment is already added in `ToolItems.xxx.get(material)`, so is no longer needed here, I'll remove the commented out code